install/pbench_agent_bootstrap.yml

```diff
@@ -0,0 +1,67 @@
+---
+#
+# Playbook for installing pbench_bootstrap
```

Comment:
Why is this named `pbench_bootstrap`?  Seems like this playbook is independent of anything pbench.

Reply:
This playbook clones the perf-dept repo, get the IP's of the computes, uperf client VM , uperf server VM and trafficgen host and generates an inventory including these host under server group and installs the necessary repos using the repo-bootstrap.yml  according to the Pbench doc